frontend/src/components/common/form/Typeahead.tsx

```diff
@@ -70,7 +70,7 @@ export function TypeaheadField<T extends TypeaheadModel>({
     setFieldTouched(name, true);
     if (
       (!getIn(name, 'value') && val !== '' && getIn(values, name) === '') ||
-      getIn(values, name) === null
+      (!getIn(values, name) && getIn(values, name) !== '')
```

Comment:
Need to make sure name is blank, other wise it will auto select 100 Mile House on blur